spec/ico_validation_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe IcoValidation do
+  subject { IcoValidation }
+
+  ['61499609', '25275500', '29233011'].each do |valid_ico|
+    it "ICO #{valid_ico} is valid" do
+      expect(subject.valid_ico?(valid_ico)).to eq true
+    end
+  end
+
+  it "ICO is valid with integer value in right format" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.